src/etc/inc/dyndns.class

```diff
@@ -702,6 +703,19 @@
 					curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, FALSE);
 					curl_setopt($ch, CURLOPT_URL, $server . '?domains=' . str_replace('.duckdns.org', '', $this->_dnsHost) . '&token=' . $this->_dnsUser);
 					break;
+				case 'googledomains':
+					if ($this->_dnsVerboseLog)
+						log_error("Google Domains: ({$this->_dnsHost}): DNS update() starting.");
+					$server = "https://domains.google.com/nic/update";
+					$post_data['hostname'] = $this->_dnsHost;
+					$post_data['myip'] = $this->_dnsIP;
+					$post_data['offline'] = 'no';
+					curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, FALSE);
```

Comment:
After further research and testing, I found that the issue is limited to installations using LibreSSL, and it looks like it has already been fixed for the next release (https://github.com/libressl-portable/portable/issues/80).

In the meantime, `CURLOPT_SSL_VERIFYPEER` can be set to true and it will work correctly for OpenSSL installations.